.github/workflows/sycl-nightly.yml

```diff
@@ -60,46 +60,63 @@ jobs:
             runner: '["Linux", "amdgpu"]'
             image_options: -u 1001 --device=/dev/dri --device=/dev/kfd
             target_devices: hip:gpu
-            tests_selector: e2e
 
-          - name: Intel L0 GPU
+          - name: NVIDIA/CUDA
+            runner: '["Linux", "cuda"]'
+            image_options: -u 1001 --gpus all --cap-add SYS_ADMIN
+            target_devices: cuda:gpu
```

Comment:
we already test on aws, do we get anything by testing on the self-hosted runner?

Reply:
Peace of mind. I want to verify all the XFAILs and the full-mode testing on our self-hosted runner.

Reply:
ok sure